### DISCUSSION TITLE
Zitadel load balance instances

### DISCUSSION THREAD

**Discussion by lanphan** [zitadel/zitadel#6198] [Q&A]
Hello,

I see that we have load balance example here https://zitadel.com/docs/self-hosting/deploy/loadbalancing-example but running it doesn't show much about load balancing.

I have some questions:
1. Can we run multiple zitadel nodes with deploy replicas > 1?
2. Instance term in Zitadel is NOT corresponding with real zitadel node, is it? I mean, I can have 3 real zitadel nodes and all 3 zitadel nodes have same 1 instance config in zitadel?

**Comment by fforootd** [comment]
Hi

You can absolutely deploy multiple processes/containers of zitadel that share the same config. Zitadel is itself stateless which means as long as the config/database is shared all of them can serve the same traffic.

A zitadel instance is a virtual zitadel that is isolated from other instances. You could call it tenant.

So the zitadel process is able to serve multiple instances (which are identified by their hostnames).

You can find more about this concept here https://zitadel.com/docs/concepts/structure/instance#multiple-virtual-instances

Hope this helps 😎
